pkg/causality/interfaces.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
I prefer to rename this file to `worker.go`, because `interfaces` means nothing.